STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -256,39 +246,55 @@ to a delegation configuration object as shown in {{fig-account-object}}.
   ],
   "termsOfServiceAgreed": true,
   "orders": "https://example.com/acme/orders/rzGoeA",
-  "delegations": [
-    "https://acme.ido.example/acme/acct/ndc/delegations/1",
-    "https://acme.ido.example/acme/acct/ndc/delegations/2"
-  ]
+  "delegations": "https://acme.ido.example/acme/delegations/adFqoz"
 }
 ~~~
 {: #fig-account-object title="Example Account object with delegations"}
 
+#### Delegation Objects
+
+This profile extends the ACME resource model with a new read-only delegation
+object that represents a delegation configuration that applies to a given NDC.
+
+A delegation object contains the CSR template (see {{sec-csr-template}}) that
+applies to that delegation, and optionally any related CNAME mapping for the
+delegated identifiers.  Its structure is as follows:
+
+- csr-template (required, object): CSR template as defined in
+  {{sec-csr-template}}.
+- cname-map (optional, object): a list of domain name pairs.  The left-hand
+  side is the delegated identifier, the right-hand side is the corresponding
+  IdO name that is aliased in the IdO's zone file to redirect the resolvers to
+  the delegated entity.  This field is only meaningful for identifiers of type
+  `dns`.
+
+An example delegation object is shown in {{fig-configuration-object}}.
+
+~~~
+{::include CSR-template/example-configuration-object.json}
+~~~
+{: #fig-configuration-object title="Example Delegation Configuration object"}
+
 In order to indicate which specific delegation applies to the requested
-certificate a new `delegation` attribute is added to the Order object on the
-NDC-IdO side (see {{sec-profile-order-journey}}).  The value of this attribute is
-the URL pointing to the delegation configuration object that is to be used for
-this certificate request.  If the `delegations` attribute in the Order object
-contains a URL that does not correspond to a configuration available to the requesting NDC, the IdO
-MUST return an error response with status code 403 (Forbidden) and type
+certificate a new `delegation` attribute is added to the identifier in the
+Order object on the NDC-IdO side (see {{sec-profile-order-journey}}).  The
+value of this attribute is the URL pointing to the delegation configuration
+object that is to be used for this certificate request.  If the `delegation`
+attribute in the Order object contains a URL that does not correspond to a
+configuration available to the requesting NDC, the IdO MUST return an error
+response with status code 403 (Forbidden) and type
 `urn:ietf:params:acme:error:unknownDelegation`.
 
 ### Order Object, the Journey from NDC to ACME Server via IdO
```

Comment:
I find this title too literary... Maybe, "Order Object Transmitted from NDC to IdO and to ACME Server".

Reply:
sure, will change as suggested